docs/Building-an-OpenSearch-and-OpenSearch-Dashboards-Distribution.md

```diff
@@ -104,18 +86,35 @@ build:
   distribution: rpm
   id: '5905'
 ```
+
+### Where to find the nightly artifacts?
+
+Below are the details where you can find the nighly artifacts build for OpenSearch and OpenSearch-Dashboards distribution. 
+Please note that since these nightly artifacts, they are not recommended for production use.
+
+#### OpenSearch-Min Snapshots
   
-#### Latest Distribution URL
-  
-Use the `latest` keyword in the URL to obtain the latest build for a given version. For example https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/2.2.0/latest/linux/x64/rpm/dist/opensearch/manifest.yml redirects to [build 5905](https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/2.2.0/5905/linux/x64/rpm/dist/opensearch/manifest.yml) at the time of writing this.
+Snapshots for OpenSearch core/min can be downloaded and used in CI's, local development, etc using below links:
   
-The `latest` keyword is resolved to a specific build number by checking an `index.json` [file](https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/2.2.0/index.json). This file has contents such as this.
+Linux:
+```
+https://artifacts.opensearch.org/snapshots/core/opensearch/<version>-SNAPSHOT/opensearch-min-<version>-SNAPSHOT-linux-x64-latest.tar.gz
+```
+Macos:
+```
+https://artifacts.opensearch.org/snapshots/core/opensearch/<version>-SNAPSHOT/opensearch-min-<version>-SNAPSHOT-darwin-x64-latest.tar.gz
+```
   
+Windows:
 ```
-{"latest":"5905"}
+https://artifacts.opensearch.org/snapshots/core/opensearch/<version>-SNAPSHOT/opensearch-min-<version>-SNAPSHOT-windows-x64-latest.zip
 ```
+
+#### Latest Distributions Builds
   
-The file is updated when a distribution build job is completed for the given product and version (or is created when such distribution job succeeds for the first time). Since one distribution build job consists of multiple stages for different combinations of distribution type, platform and architecture, the `index.json` is only modified once all stages succeed. With this said, the `latest` url only works when the distribution build job succeeds at least once for the given product and version.
+Use the `latest` keyword in the URL to obtain the latest nightly build for a given version. For example https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/2.2.0/latest/linux/x64/rpm/dist/opensearch/manifest.yml redirects to [build 5905](https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/2.2.0/5905/linux/x64/rpm/dist/opensearch/manifest.yml) at the time of writing this.
+  
+The file is updated for each distribution, pltaform and architecture separately. Hence the `latest` number would vary.
```

Comment:
Hey Sayali, can you update the above line to this to make it more clear.
```
The `latest` keyword is resolved to a specific build number by checking an `index.json` [file](https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/2.12.0/index/linux/x64/tar/index.json). This file is maintained individually for each platform, architecture and distribution. The index.json file has contents similar to
{"latest":"9445"}

```

Reply:
Done